packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1670,7 +1670,7 @@ For brevity, the following response doesn't include all available properties.
     },
     "name": {
       "title": "Name",
-      "description": "A name for this integration instance in Okta",
+      "description": "A name for this integration log stream in Okta",
```

Comment:
I'm not sure of the structure being described here. It might be possible that one of the following might be more suitable:

- A name for this integration's log stream in Okta.
- A name for this log stream in Okta.
- The name to use for this integration's log stream in Okta.

Reply:
ah, right, good catch, I updated it now to option 2